proj/src/main/webapp/script.js

```diff
@@ -66,7 +67,23 @@ function initRef() {
 function pushChatMessage() {
     const messageRef = firebase.database().ref(path);
```

Comment:
I think constant variables like `path` here are usually defined in all caps. At least this is true for python.

Reply:
fixed!